syntax = "proto3";

service WineQualityPredictor {
    rpc PredictQuality (PredictRequest) returns (PredictResponse) {}
}
  
message PredictRequest {
  float fixed_acidity = 1;
  float volatile_acidity = 2;
  float citric_acid = 3;
  float residual_sugar = 4;
  float chlorides = 5;
  float free_sulfur_dioxide = 6;
  float total_sulfur_dioxide = 7;
  float density = 8;
  float ph = 9;
  float sulphates = 10;
  float alcohol = 11;
}
  
message PredictResponse {
  float quality = 1;
}
